core/src/main/java/curso/exception/CursoIncompletoException.java

```diff
@@ -0,0 +1,7 @@
+package curso.exception;
+
+public class CursoIncompletoException extends RuntimeException {
+    public CursoIncompletoException(String s) {
```

Comment:
No es crítico, pero cambiar "s" por "message" o similar
```suggestion
    public CursoIncompletoException(String message) {
```